cypress/component/utils/bucket_utils.spec.ts

```diff
@@ -16,12 +46,90 @@ const dar_collection = {
       id: 1,
       referenceId: 'dar-reference-id-1',
       collectionId: 1,
+      data: {
+        projectTitle: 'Test Project',
+        piName: 'Test PI Name',
+        darCode: 'DAR-001',
+        checkNihDataOnly: false,
+        rus: '',
+        nonTechRus: '',
+        diseases: false,
+        methods: false,
+        controls: false,
+        population: false,
+        other: false,
+        otherText: '',
+        forProfit: false,
+        oneGender: false,
+        gender: '',
+        pediatric: false,
+        illegalBehavior: false,
+        addiction: false,
+        sexualDiseases: false,
+        stigmatizedDiseases: false,
+        vulnerablePopulation: false,
+        populationMigration: false,
+        psychiatricTraits: false,
+        notHealth: false,
+        hmb: false,
+        status: 'submitted',
+        poa: false,
+        datasets: [],
+        restriction: {},
+        validRestriction: true,
+        researchPlans: 'Test research plans',
+        anvilUse: false,
+        cloudUse: false,
+        localUse: false,
+        cloudProvider: '',
+        cloudProviderType: '',
+        cloudProviderDescription: '',
+        geneticStudiesOnly: false,
+        irb: false,
+        itDirector: '',
+        itDirectorEmail: '',
+        signingOfficial: '',
+        signingOfficialEmail: '',
+        publication: false,
+        collaboration: false,
+        forensicActivities: false,
+        sharingDistribution: false,
+        ontologies: [],
+        labCollaborators: [],
+        internalCollaborators: [],
+        externalCollaborators: [],
+        dsAcknowledgement: false,
+        gsoAcknowledgement: false,
+        pubAcknowledgement: false,
+        piEmail: 'test@example.com',
+        piCountryOfOperation: 'US',
+      },
+      draft: false,
+      progressReport: false,
+      expired: false,
+      rationale: '',
+      rus: {},
+      status: 'submitted',
+      userId: 1,
+      eraCommonsId: 'test-era-commons-id',
+      expiresAt: new Date().getTime(),
+      createDate: new Date().getTime(),
+      sortDate: new Date().getTime(),
+      submissionDate: new Date().getTime(),
+      updateDate: new Date().getTime(),
```

Comment:
`new Date().getTime()` could be a constant which would simplify a lot of usages in simple object construction. 

Separately, it is very concerning to me that so much additional code is required solely due to updates to `model.ts`. Locally, I reverted all of this code and the tests run with no changes. So much boilerplate feels like an anti-pattern to me. I wonder if it's time to think about genericizing obect construction to a cypress helper class.